src/libraries/System.Private.CoreLib/src/System/Diagnostics/Tracing/RuntimeEventSourceHelper.Windows.cs

```diff
@@ -25,7 +25,7 @@ internal static int GetCpuUsage()
                 if (s_prevSystemUserTime != 0 && s_prevSystemKernelTime != 0 && // These may be 0 when we report CPU usage for the first time, in which case we should just return 0.
                     totalSystemTime != 0)
                 {
-                    cpuUsage = (int)(totalProcTime * 100 / totalSystemTime);
+                    cpuUsage = (double)(totalProcTime * 100 / totalSystemTime);
```

Comment:
```suggestion
                    cpuUsage = totalProcTime * 100.0 / totalSystemTime;
```

The expression (totalProcTime * 100 / totalSystemTime) already rounds to an integer value so casting to a double afterwards has no effect. In order to do floating point division at least one of the numerator or denominator needs to be a floating point value. For example:
```C#
Console.WriteLine((double)(1 * 100 / 500));    // prints 0
Console.WriteLine((double)(1 * 100.0 / 500));  // prints 0.2
```